specs/src/specs/figures/sparse_start_share.dot

```diff
@@ -7,7 +7,7 @@ digraph G {
             <td align="left" border="0" cellpadding="0">0</td>
             <td align="left" border="0" cellpadding="0">8</td>
             <td align="left" border="0" cellpadding="0">9</td>
-            <td align="left" border="0" cellpadding="0">12</td>
+            <td align="left" border="0" cellpadding="0">13</td>
```

Comment:
unrelated comment

if we pad this to 16, zk rollups could save quite a few cycles, correct? could they do that themselves?

Reply:
@preston-evans98 commented that aligning to 4 or 8 byte boundaries would help their use-case. I assume this is for performance reasons but if that's the case, we may want benchmarks to make an informed decision on how best to pad / align the universal share prefix.

Reply:
cool, 👍